### PR TITLE
Feature/prime 1050 add org info to site reg review

### PR DESCRIPTION
**Pull request by hogmann**

### CODE REVIEW

File: prime-dotnet-webapi/Views/SiteRegistrationReview.cshtml [hunk 1]
```diff
@@ -287,6 +287,65 @@
...
+  </div>
+
+  <div class="ml-2 mb-2">
+    <h5>Date of Birth</h5>
+    <span class="ml-2">@Model.Organization.SigningAuthority.DateOfBirth</span>
+  </div>
+
+  <div class="ml-2 mb-2">
+    <h5>Country</h5>
+    <span class="ml-2">@Model.Organization.SigningAuthority.PhysicalAddress.Country</span>
```
Comment (AnaisHebert) [inline prime-dotnet-webapi/Views/SiteRegistrationReview.cshtml:304]: Should be either PhysicalAddress.CountryCode or PhysicalAddress.Country.Name

File: prime-dotnet-webapi/Views/SiteRegistrationReview.cshtml [hunk 2]
```diff
@@ -287,6 +287,65 @@
...
+  </div>
+
+  <div class="ml-2 mb-2">
+    <h5>Province</h5>
+    <span class="ml-2">@Model.Organization.SigningAuthority.PhysicalAddress.Province</span>
```
Comment (AnaisHebert) [inline prime-dotnet-webapi/Views/SiteRegistrationReview.cshtml:309]: Should be PhysicalAddress.ProvinceCode

**Review by AnaisHebert — APPROVED** [review]
🎈🎈

**Review by james-hollinger — APPROVED** [review]
👍👍👍

**Pull request merged**